x-pack/test/alerting_api_integration/common/fixtures/plugins/aad/index.ts

```diff
@@ -52,7 +52,7 @@ export default function(kibana: any) {
             namespace,
           });
           return { success: true };
-        },
+        }) as Hapi.Lifecycle.Method,
```

Comment:
Hapi doesn't allow to define a type for request payload https://github.com/DefinitelyTyped/DefinitelyTyped/issues/25605

Reply:
I did not see `hapi` package upgrade in `package.json`. Why was these `Hapi.Lifecycle.Method` additions needed?

Reply:
no idea to be honest, I suspect adding `h2o2` & `inert` types affects `hapi` somehow, since they both extend `hapi` interfaces.